test/OrchardCore.Tests/Data/ContentItemTests.cs

```diff
@@ -46,9 +47,7 @@ public void ShouldSerializeParts()
         [Fact]
         public void ShouldUpdateContent()
         {
-            var contentItem = new ContentItem();
-            contentItem.GetOrCreate<MyPart>();
-            contentItem.Alter<MyPart>(x => x.Text = "test");
+            var contentItem = CreateContentItemWithPart();
```

Comment:
I personally like code duplication when it's small, it talks for itself, doesn't need me to jump to some other places or if I am lazy not understand what it actually does.

Reply:
Counter argument: If you create a good method name, the name should tell you what it does without having to duplicate code or guess

Reply:
Agreed. In that case, and because I don't use this API much, it felt good to understand what it meant to create a content item and add a part. And then the statements after with .Alter had more meaning in this context

Reply:
maybe rename `CreateContentItemWithPart` to `CreateContentItemWithMyPart`

Reply:
When I was about to add a test, knowing if each test's setup section was identical or not was more important than the specifics. I had to stop and compare methods, which I didn't enjoy. So it wasn't just DRY for DRY's sake.